src/test/java/api/pet/Post_NegativeTest.java

```diff
@@ -0,0 +1,246 @@
+package api.pet;
+
+import api.apiwrappers.PetApiWrapper;
+import api.domain.pets.Category;
+import api.domain.pets.Pet;
+import api.domain.pets.PetStatus;
+import api.domain.pets.Tags;
+import api.utils.*;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.apache.commons.lang3.RandomUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.HttpStatus;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+
+public class Post_NegativeTest {
+    private static final PetApiWrapper petApiWrapper = new PetApiWrapper();
+    private static Pet petPayload;
+    private static Category category = new Category();
+    private static Tags tag = new Tags();
+    private static ArrayList<Tags> tags = new ArrayList<>();
+
+    @BeforeClass
```

Comment:
here should be BeforeMethod, you need new payload on each test

Reply:
Changed to "@BeforeMethod".